app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Here is the details of user</h1>
\ No newline at end of file
+<section class="my-bio bg-secondary bg-gradient">
+  <div class="container">
+    <div class="row">
+      <%= render partial: 'user', locals: { user: @user } %>
+    </div>
+    <div class="card my-4">
+      <%= render partial: 'bio', locals: { user: @user } %>
+    </div>
```

Comment:
- [ ] You are making use of a partial here but the partial file is missing and it's causing the app to crash as a result of not locating the partial template being passed here. Kindly fix that in this file and the others 👍🏽

<img width="1158" alt="Screenshot 2023-11-03 at 10 30 13 PM" src="https://github.com/maths29/iwuBlog/assets/50941074/a4c15f28-37dd-429d-b96c-da71d6c112da">